src/main/java/frc/robot/subsystems/elevatorIO/Elevator.java

```diff
@@ -23,7 +24,8 @@ public Elevator(ElevatorIO IO) {
     // this.elevatorController = new ProfiledPIDController(0.6, 0, 0.2, new Constraints(160, 500));
     this.feedforward = new ElevatorFeedforward(0, 5.212, 0);
     // this.elevatorController = new ProfiledPIDController(0.15, 0, 0.1, new Constraints(160, 500));
-    this.elevatorController = new ProfiledPIDController(0.6, 0, 0.2, new Constraints(160, 500));
+    // this.elevatorController = new ProfiledPIDController(0.6, 0, 0.2, new Constraints(160, 500));
+    this.elevatorController = new ProfiledPIDController(0.0, 0, 0.0, new Constraints(160, 500));
```

Comment:
is no gains fine? feels like something that could cause problems while testing if we forget about it, idk

Reply:
fixed